### ISSUE TITLE
Laravel auto instrumentation does not support semconv 1.27, yet it is not locked to 1.26 through composer.json

### ISSUE THREAD

**Issue by LauJosefsen** [open-telemetry/opentelemetry-php#1367]
Hi.

Updating to the latest semconv release produces this warning for queue workers for the latest version of the auto instrumentation for laravel.

```
PHP Warning: Illuminate\Queue\Worker::getNextJob(): OpenTelemetry: pre hook threw exception, class=Illuminate\Queue\Worker function=getNextJob message=Undefined constant OpenTelemetry\SemConv\TraceAttributeValues::MESSAGING_OPERATION_RECEIVE in Unknown on line 0
```

It looks like the [auto instrumentation for Laravel depends on the SemConv constant MESSAGING_OPERATION_RECEIVE](https://github.com/open-telemetry/opentelemetry-php-contrib/blob/main/src/Instrumentation/Laravel/src/Hooks/Illuminate/Queue/Worker.php#L102) which was removed in [the semconv 1.27 release yesterday.](https://github.com/opentelemetry-php/sem-conv/releases/tag/1.27.0)

If minor bumps of semconv can contain breaking changes, I propose locking the version in the composer.json of the individual auto instrumentations.

**Comment by ChrisLightfootWild** [comment]
I can confirm the behavior here too. I don't think we should lock to specific versions but should have perhaps kept that constant to prevent breakages.

**Comment by brettmc** [comment]
Looks like it was in 1.26 (which never got tagged): https://github.com/open-telemetry/opentelemetry-php/commit/a822c455a45e2ea372dc665be26cfb7ef80415db

I've asked on the semconv list about this. Attributes being dropped has historically been a problem, and part of our process is to check for missing trace+resource attributes, however not the values (probably because it's never caused an issue before). The workaround is probably to locate the removed values and replace them as deprecated (like we do when TraceAttributes go missing), and release that as 1.27.1 - I'll wait a day or so to see whether anyone from semconv group has any insight.

**Comment by brettmc** [comment]
Seems it was also a surprise to the SemConv team, and they'll add some more checks to try to avoid this in future. They also pointed out it's an experimental attribute, so is subject to change.

https://github.com/open-telemetry/opentelemetry-php/pull/1368 adds back those values (and I didn't find any more removals when looking through the diffs for 1.26 and 1.27 changes)

**Comment by brettmc** [comment]
Have just released SemConv 1.27.1 - can you check it out, @LauJosefsen ?

**Comment by firecow** [comment]
@brettmc Our services, after upgrade to open-telemetry/sem-conv=1.27.1 has stopped producing the error logs.
@LauJosefsen and I work on the same projects.

Guess when I deployed 1.27.1 :smile: 

![image](https://github.com/user-attachments/assets/d361b366-9eea-4144-a960-fdddf3450857)
